src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -80,9 +80,24 @@ public class RepositoryCloneViewModel : BaseViewModel, IRepositoryCloneViewModel
                 loading => loading.Value || repositories.UnfilteredCount > 0 && !LoadingFailed)
                 .ToProperty(this, x => x.FilterTextIsEnabled);
 
-            this.WhenAny(x => x.IsLoading, x => x.LoadingFailed,
-                (loading, failed) => !loading.Value && !failed.Value && repositories.UnfilteredCount == 0)
-                .Subscribe(x => NoRepositoriesFound = x);
+            this.WhenAny(
+                x => x.repositories.UnfilteredCount,
```

Comment:
Tests are broken, and I suspect it may be because `WhenAny` is initially triggered with null values (or some such weirdness). It's weird, `Repositories` is being set before this, but it's very likely this access blowing up.

Probably best to do something like

```
this.WhenAny(x => x.IsLoading, x => x.LoadingFailed, x => x.repositories?.UnfilteredCount,
    (loading, failed, count) => !loading.Value && !failed.Value && count != null && count.Value == 0)
    .Subscribe(x => NoRepositoriesFound = x);
```